create-liff-app.ts

```diff
@@ -204,7 +194,7 @@ const questions: Array<Question | ListQuestion> = [
         console.log(`\n${chalk.yellow('The project is already exists.')}`);
         return false;
       }
-      
+
```

Comment:
Could you remove this unnecessary change?

Reply:
ok, I think that was removed by the formatter.